src/lib/navigation.ts

```diff
@@ -0,0 +1,11 @@
+import {createNavigationContainerRef} from '@react-navigation/native';
```

Comment:
This navigation ref, allows you to call `navigate` outside of the react-tree. Its useful for tests.